.github/workflows/kata-deploy-push.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
 
       - name: Build ${{ matrix.asset }}
         run: |
+          sudo apt-get install -y git git-extras libipc-run-perl libssl-dev pkg-config protobuf-compiler
```

Comment:
ooi, why is `git-extras` needed here and below?

Reply:
I can remove `git-extras` in kata-deploy-push.yaml which is new added, but in snap-release.yaml which already have it before my pr

Reply:
Samuel found we need add the dependency in docker file, this change will be dropped